### PR TITLE
R10k gem

### PR DESCRIPTION
**Pull request by tuxmea**
readd r10k gem. this allows local testing without any puppet installation by running bundle exec r10k puppetfile install

### CODE REVIEW

**Comment by tuxmea** [comment]
wrong. one commit to much

**Comment by alvagante** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          3 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">There’re changes in manifests, but not tests. That’s OK as long as you’re refactoring existing code.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Unless you’re refactoring existing code, please update CHANGELOG.md.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR! Big changes, big things may happen! Check them.</td>
    </tr>
  </tbody>
</table>

Here's an example of a CHANGELOG.md entry:

```markdown
* [#176](https://github.com/example42/psick/pull/176): R10k gem - [@tuxmea](https://github.com/tuxmea).
```


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: site/profile/manifests/puppet/pe_agent.pp [hunk 1]
```diff
@@ -1,8 +1,13 @@
 # This class manages tp::test for PE Agents
 #
 class profile::puppet::pe_agent (
-  Boolean $test_enable = false,
-  Hash $settings       = {},
+  Boolean $test_enable        = false,
```
Comment (alvagante) [inline site/profile/manifests/puppet/pe_agent.pp]: Please make relevant MR to puppet-psick, here as in the other classes in old profile

File: site/tools/manifests/user/managed.pp [hunk 2]
```diff
@@ -123,6 +124,17 @@
     create_resources(ssh_authorized_key, $sshkeys_content, $sshkey_defaults)
   }
 
+  if $generate_ssh_keypair {
+    if $id_rsa_source == '' and $id_rsa_pub_source == '' {
+      tools::ssh_keygen { $name:
```
Comment (alvagante) [inline site/tools/manifests/user/managed.pp]: FYI. I don't know if this comes for a change of yours or something already in old code somehow shown here, in any case note that both the old profile and the tools modules when migrated to the single psick module. Defines were renamed from tools::ssh_keygen to, in this case, psick::ssh::keygen.

**Pull request closed without merging**